### DISCUSSION TITLE
Are albums created/preserved?

### DISCUSSION THREAD

**Discussion by daschillinger** [simulot/immich-go#127] [Q&A] [answered]
I did a immich-go on a google takeout like this:
./immich-go -server=http://mynas:2283 -key=zzV6k65KGLNB9mpGeri9n8Jk1VaNGHSCdoH1dY8jQ upload
-google-photos ~/Download/takeout-*.zip
I was leaving away "-create-albums" as the documentation said "Controls creation of Google Photos albums in Immich (default TRUE)."
so I thought I do not have to set it
the process is running but until now I see no albums.

**Comment by simulot** [comment, marked as answer]
I confirm, the create album option is activated by default. 
Albums are created at the very end of the import task.

**Reply by daschillinger** [reply]
could you answer another noob-question?
if it fails(sorry don't remember the error code) and I try again and again and some time it will be finished, 
are the photos retried which failed before? and maybe I am still getting a complete upload?
atm I wait for a new takeout as my old one was from 2022 and I think maybe google changed something in the meanwhile

**Reply by simulot** [reply]
You can run the import as you want. Already passed photos will be ignored.

**Reply by daschillinger** [reply]
Server error             : Takeout/Google Fotos/Photos from 2019/IMG_0352.MP4: AssetUpload, POST, http://192.168.178.80:2283/api/asset/upload, 400 Bad Request
Bad Request 
it stands there a while then goes on
